cmd/nerdctl/build.go

```diff
@@ -359,9 +364,30 @@ func generateBuildctlArgs(cmd *cobra.Command, buildkitHost string, platform, arg
 	if err != nil {
 		return "", nil, false, "", nil, nil, err
 	}
-	if _, err := os.Lstat(filepath.Join(absDir, file)); err != nil {
-		return "", nil, false, "", nil, nil, err
+	if file != buildkitutil.DefaultDockerfileName {
+		if _, err := os.Lstat(filepath.Join(absDir, file)); err != nil {
+			return "", nil, false, "", nil, nil, err
+		}
+	} else {
+		dockerfile, dockerfileerr := os.ReadFile(filepath.Join(absDir, file))
+		containerfile, containerfileerr := os.ReadFile(filepath.Join(absDir, Containerfile))
+		// both files exist.
+		if dockerfileerr == nil {
+			if containerfile == nil && !bytes.Equal(dockerfile, containerfile) {
+				cmd.OutOrStdout().Write([]byte("Dockerfile and Containerfile have different contents, building with Dockerfile"))
```

Comment:
Why not logrus ?

Reply:
I added to logrus. @AkihiroSuda would you prefer having it printed out to stdout as well?

Reply:
logrus should be enough?

Reply:
Okay! Thanks! This change only shows with logrus